style.css

```diff
@@ -0,0 +1,427 @@
+  }
+
+  .toolbar {
+    flex: 1 1 auto;
+    display: flex;
+    height: 48px;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 40px;
+  }
+
+  .logo-place {
+    margin-left: 12px;
+  }
+
+  .my-logo {
+    padding: 10px 12px 10px 12px;
+    font-family: sans-serif, "Poppins";
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+    text-decoration: none;
+  }
+
+  .headline {
+    width: 100%;
+    background-image: url("images/Header\ bg.png");
+    background-size: 380px 550px;
+    border-radius: 0 0 0 100px;
+    display: flex;
+    flex-direction: column;
+    background-color: #fff;
+    padding: 20px 10px 0 10px;
+    height: calc(718px - 60px);
+    background-repeat: no-repeat;
+    overflow: hidden;
+  }
+
+  .headline > * {
+    padding-left: 20px;
+    padding-right: 20px;
+  }
+
+  .headline h1 {
+    font-family: sans-serif, "Poppins";
+    font-style: normal;
+    font-weight: 700;
+    font-size: 30px;
+    line-height: 35px;
+    letter-spacing: 0.25px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    padding-top: 80px;
+  }
+
+  .text {
+    font-family: sans-serif, "Poppins";
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    text-align: justify;
+    text-justify: inter-word;
+    width: auto;
+    height: 150px;
+    padding-right: 20px;
+  }
+
+  .connect {
+    font-family: sans-serif, "Poppins";
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .social-media {
+    display: flex;
+    width: 170.17px;
+    height: 20px;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: baseline;
+    padding-bottom: 136px;
+    list-style-type: none;
+    padding-left: 15px;
+  }
+
+  .works {
+    display: grid;
+    grid-template-columns: auto;
+  }
+
+  .grid-items {
+    display: flex;
+    flex-direction: column;
+    align-items: left;
+    padding: 16px;
+    gap: 12px;
+    background: #fff;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    margin-left: 24px;
+    margin-right: 24px;
+  }
+
+  .project-1 {
+    margin-top: 114px;
+  }
+
+  .other-projects {
+    margin-top: 88px;
+  }
+
+  .project-img {
+    border-radius: 8px;
+    width: auto;
+  }
+
+  .details {
+    width: auto;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .details h3 {
+    margin: initial;
+    padding: initial;
+  }
+
+  .details ul {
+    margin: initial;
+    padding: initial;
+  }
+
+  .details ul li {
+    margin: initial;
+    padding: initial;
+  }
+
+  .titles {
+    display: flex;
+    flex-direction: column;
+    gap: 12px;
+    flex-wrap: wrap;
+  }
+
+  .titles p {
+    color: #344563;
+  }
+
+  .project-title {
+    font-family: sans-serif, "Poppins";
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
```

Comment:
You have the correct font set up for the cards but the way it is being implemented is wrong. Kindly use Poppins first before san-serif because doing it as you have done makes sans-serif the first font that will be applied and Poppins the fallback.
Kindly revert the order here and in all other areas where this mistake is made in order to make your design consistent with the Figma template.

```
  font-family: "Poppins", sans-serif;
```